PKHeX.Core/Saves/Substructures/Gen4/Ranch/RanchToyType.cs

```diff
@@ -30,5 +30,21 @@ public enum RanchToyType : byte
     Sprint_Stand = 22,
     Attractor = 23,
     Challenger = 24,
-    Toy_Box = 25,
+    Toy_Box = 25, // Supports metadata - Metadata value is the index of the contained toy.
+    Poke_Ball = 26, // Normally unused
```

Comment:
I've got a thread open about this one, because I couldn't find any mention of its existence online:
https://projectpokemon.org/home/forums/topic/62995-my-pok%C3%A9mon-ranch-unused-pok%C3%A9-ball-poketoy/